yaml/network-policy/edge-router-net-policy.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: edge-router
+  namespace: openfaas
+spec:
+  policyTypes:
+    - Ingress
+  podSelector:
+    matchLabels:
+      app: edge-router
+  ingress:
+  - from:
+    - namespaceSelector: {}
+      podSelector:
+        matchLabels:
+          app.kubernetes.io/name: ingress-nginx
```

Comment:
Is it possible to keep both for legacy users? ingress-nginx and nginx-ingress?

Reply:
re-added the old label
https://github.com/openfaas/openfaas-cloud/pull/667/commits/95ac2e4ef2dd4b458322d8205f74d5a433c6b085